components/seo-layout/page-layout-with-seo.component.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Head from 'next/head';
+
+type PageLayoutWithSEOProps = {
+  title: string;
+  description: string;
```

Comment:
```
type MetaProps = {
  description: string;
}

type PageLayoutWithSEOProps = {
  title: string;
  meta: MetaProps
}
```

Please scope description to meta tag. I believe I gave the same feedback earlier as well.